src/state.js

```diff
@@ -0,0 +1,27 @@
+import _ from 'lodash/fp'
+import * as F from 'futil-js'
+
+export let defaultState = {
+  path: null,
+  updating: null,
+  lastUpdateTime: null,
+  markedForUpdate: null,
+  hasValue: null,
+  error: null,
+  context: null,
+  missedUpdate: null,
+  pause: null,
+}
```

Comment:
since we have this list, we should consider reusing this in serialize because we have a list of what keys we should exclude (with `lastUpdateTime` being the one exception)